RevenueCatUI/Templates/V2/ViewHelpers/BackgroundStyle.swift

```diff
@@ -16,7 +16,7 @@ import SwiftUI
 
 #if PAYWALL_COMPONENTS
 
-enum BackgroundStyle {
+enum BackgroundStyle: Hashable {
```

Comment:
Added a bunch of `Hashable`s to be able to programmatically create previews of all combinations using `ForEach`.